typespec-tests/src/main/java/com/payload/multipart/MultiPartAsyncClient.java

```diff
@@ -151,7 +154,10 @@ Mono<Response<Void>> jsonPartWithResponse(BinaryData body, RequestOptions reques
      * {
      *     id: String (Required)
      *     pictures (Required): [
-     *         byte[] (Required)
+     *         BinaryData (Required)
+     *     ]
+     *     pictures (Optional): [
```

Comment:
The two properties are all called `pictures`?

Reply:
The other one if "picturesFilenames" we appended during codegen.

This method will not get exposed, so user should not able to see this javadoc.

Reply:
But this reminds me of the "additionalProperties" case, that the Javadoc is exposed and be wrong...
https://github.com/Azure/autorest.java/blob/main/typespec-tests/src/main/java/com/type/property/additionalproperties/IsFloatClient.java#L44-L51

I will had a follow up task to say e.g. this ModelProperty is not via JSON etc.

Reply:
ok, I see.

Reply:
Another "fix" for avoiding the confusing multipart schema is to just "do not generate Javadoc if the method is not public or protected". It could save some file space as we have this kind of very long Javadoc. But that likely be another round of discussion.

Reply:
How about first remove the javadoc for the package-private method for multipart only? Because the schema for multipart is meaningless

Reply:
OK, I've skip the body request schema, if it is multipart.